templates/tweet/index.html

```diff
@@ -1 +1,172 @@
-{% extends "includes/base.html" %}
\ No newline at end of file
+{% load static %}
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Twitter</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        {% block specific_meta %}
+
+        {% endblock %}
+
+
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
+        <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
No bootstrap or any fancy library. Use plain html css.

Reply:
removed and a simple css file added for basic design